src/bin/rav1e.rs

```diff
@@ -265,6 +265,31 @@ fn init_logger() {
   pretty_env_logger::init();
 }
 
+cfg_if::cfg_if! {
+  if #[cfg(target_os = "windows")] {
+    fn print_rusage() {
+      eprintln!("Windows is not supported.");
+    }
+  } else {
+    fn print_rusage() {
+      let (utime, stime, maxrss) = unsafe {
+        let mut usage = std::mem::zeroed();
+        let _ = libc::getrusage(libc::RUSAGE_SELF, &mut usage);
+        (usage.ru_utime, usage.ru_stime, usage.ru_maxrss)
+      };
+      eprintln!(
+        "utime: {} us",
```

Comment:
I assume this stands for "user time" and "system time"? It would be clearer to print out the full words.

Reply:
Sure